packages/devtools_app/lib/src/shared/connected_app.dart

```diff
@@ -36,62 +36,62 @@ class ConnectedApp {
   Future<bool> get isFlutterApp async => _isFlutterApp ??=
       await serviceManager.libraryUriAvailable(flutterLibraryUri);
 
-  bool get isFlutterAppNow {
+  bool? get isFlutterAppNow {
```

Comment:
this can be non-nullable

Reply:
it cannot, because we override it in descendant OfflineConnectedApp, where it is nullable